backend/djangoindia/db/migrations/0005_auto_20240819_0211.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 4.2.5 on 2024-08-18 20:29
+
+import datetime
+from django.db import migrations, models
+import django.utils.timezone
+
+
+def add_created_at_value(apps, schema_editor):
+    contactus = apps.get_model('db', 'contactus')
+    event = apps.get_model('db', 'event')
+    eventregistration = apps.get_model('db', 'eventregistration')
+    newslettersubscription = apps.get_model('db', 'newslettersubscription')
+
+    contactus.objects.update(created_at=django.utils.timezone.now())
+    event.objects.update(created_at=django.utils.timezone.now())
+    eventregistration.objects.update(created_at=django.utils.timezone.now())
+    newslettersubscription.objects.update(created_at=django.utils.timezone.now())
```

Comment:
Why are you updating them using migrations when they already have `default=django.utils.timezone.now` ? 🤔